routing/routing.xml

```diff
@@ -881,6 +881,10 @@
 				<select t="hgv" v="delivery" value="0.15"/>
 			</if>
 
+            <select value="1.1" t="smoothness" v="excellent"/>
```

Comment:
They will override all other settings, so here we mention on top only disqualifying settings like < 0.4. 

Please Keep in mind all the values are not multiplied but only the top one is selected